fido2.cabal

```diff
@@ -43,12 +43,18 @@ library
     x509,
     scientific,
     vector,
-    QuickCheck
+    QuickCheck,
+    jwt,
+    http-client,
+    http-types,
+    http-client-tls,
```

Comment:
Just FYI: http-client-tls depends on cryptonite too.

If you don't want that as a dependency you should check out http-client-openssl

Reply:
We're most likely going with cryptonite after all, it's just embedded so deeply into all kinds of libraries that it would be a lot of effort to get around it completely. So why not opt into it fully.

Reply:
Sounds good.